web/rootfs/defaults/system-config.js

```diff
@@ -22,7 +22,7 @@ config.focusUserJid = 'focus@{{$XMPP_AUTH_DOMAIN}}';
 
 {{ if $ENABLE_SUBDOMAINS -}}
 var subdir = '<!--# echo var="subdir" default="" -->';
-var subdomain = "<!--# echo var="subdomain" default="" -->";
+var subdomain = '<!--# echo var="subdomain" default="" -->';
```

Comment:
I'm curious what impact you saw from this?  

The nginx container shoudl be replacing the full contents from <!-- all the way to --> using SSI directives before any javascript is parsed by the browser.  So there should not be any point at which the quotes confliect.

Reply:
Currently setting up a dev environment to make some changes on the jitsi meet server code and was struggling to load the config and though maybe its this, but just found out it was something different. So it is no bug only style and it is now the same as in the jitsi-meet repo:

https://github.com/jitsi/jitsi-meet/blob/master/config.js

Reply:
Excellent, I'm all for consistency but I just wanted to ensure I understood the impact of the existing code.